prac_05/wimbledon.py

```diff
@@ -0,0 +1,54 @@
+"""
+Wimbledon
+Estimate: I forgot to give a estimate :(
+Actual: 20 Mins
+"""
+
+champion_name_to_titles = {}
```

Comment:
From what I've seen, you have used global variables throughout this program instead of passing parameters into functions and returning them. This has been taught as a big no-no and not a good habit to get into.

Reply:
Yup, my bad I had originally done the program before refactoring the functions and forgot that i had left the global variables there, thanks for the feedback :)